APP/app.test.js

```diff
@@ -34,14 +34,33 @@ describe("GET /api/topics", () => {
 });
 
 describe("GET /api/", () => {
-    it("Should respond with an object describing all the available endpoints on the API", () => {
-      return request(app)
-        .get("/api/")
-        .expect(200)
-        .then(({ body }) => {
-         expect(body).toEqual(descriptions)
-        });
-    });
+  it("Should respond with an object describing all the available endpoints on the API", () => {
+    return request(app)
+      .get("/api/")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(descriptions);
+      });
   });
-  
+});
 
+describe("GET /api/articles", () => {
```

Comment:
I would add the. :id to the end of this description to show it's a parametric endpoint